.circleci/config.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             sudo apt install libspatialindex-dev xdg-utils
       - python/install-packages:
           pkg-manager: poetry
-          args: "-E graph_func -E viz"
+          args: "-E graph_func -E viz --with docs"
```

Comment:
I think we can truncate this to `--with graph_func,viz,docs`

Reply:
FYI idk why, but that doesn't work. My build gets an error.